website/blog/2021-08-25-webdriverio-meet-serenity-js.md

```diff
@@ -0,0 +1,52 @@
+---
+title: WebdriverIO, meet Serenity/JS
+author: Jan Molak
+authorURL: https://www.linkedin.com/in/janmolak/
+authorImageURL: https://avatars.githubusercontent.com/u/1089173?s=400&v=4
+---
+
+## WebdriverIO, meet Serenity/JS!
+
+Wouldn't it be great if you could use WebdriverIO to create full-stack test automation frameworks that interact with any interface of your system, generate in-depth reports, and help you write test code that's easy to understand, maintain, and reuse across projects and teams?
```

Comment:
Can you use WebdriverIO for this already? I believe Serenity as oppose to WebdriverIO has a more business focused approach using the Screenplay pattern for user-centred acceptance. Let's highlight this a bit more.

Reply:
Maybe have a sentence as to why would someone pick Serenity over pure WebdriverIO?